services/src/main/scala/cromwell/pipeline/service/RunService.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.pipeline.service
+
+import java.util.Date
+
+import cromwell.pipeline.datastorage.dao.repository.RunRepository
+import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.model.wrapper.RunId
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class RunService(runRepository: RunRepository)(implicit executionContext: ExecutionContext) {
+
+  def addRun(runCreateRequest: RunCreateRequest): Future[RunId] = {
+
+    val newRun = Run(
+      runId = RunId.random,
+      projectId = runCreateRequest.projectId,
+      projectVersion = runCreateRequest.projectVersion,
+      status = Created,
+      timeStart = new Date().toInstant,
+      userId = runCreateRequest.userId,
+      results = runCreateRequest.results
+    )
+
+    runRepository.addRun(newRun)
+  }
+
+  def getRunById(runId: RunId): Future[Option[Run]] = runRepository.getRunById(runId)
```

Comment:
Shouldn't we forbid to access `Run` if current user is not an author of this `Run`?

Reply:
I added the `userId` to the method